packages/editor/CodeMirror/pluginApi/codeMirrorRequire.ts

```diff
@@ -21,12 +24,14 @@ const libraryNameToPackage: Record<string, any> = {
 	'@codemirror/language': codeMirrorLanguage,
 	'@codemirror/autocomplete': codeMirrorAutocomplete,
 	'@codemirror/commands': codeMirrorCommands,
-	'@codemirror/highlight': lezerHighlight,
```

Comment:
While [`@codemirror/highlight` exists, it's deprecated and split into `@lezer/highlight` and `@codemirror/language`](https://www.npmjs.com/package/@codemirror/highlight) and thus should not be used.